### PR TITLE
Refactor JPA FAT Package Names so they're less compacted.

### PR DESCRIPTION
**Pull request by jgrassel**
Refactoring JPA FAT package names which show up on junit reports, in order to decompact them on Jenkins results.

### CODE REVIEW

**Comment by jgrassel** [comment]
#build

File: dev/com.ibm.ws.jpa_eclipselink_fat/fat/src/com/ibm/ws/jpa/FATSuite.java [hunk 1]
```diff
@@ -17,6 +17,25 @@
...
+import com.ibm.ws.jpa.eclipselink.TestOLGH8294_EJB;
+import com.ibm.ws.jpa.eclipselink.TestOLGH8294_Web;
+import com.ibm.ws.jpa.eclipselink.TestOLGH8461_EJB;
+import com.ibm.ws.jpa.eclipselink.TestOLGH8461_Web;
+import com.ibm.ws.jpa.eclipselink.TestOLGH8950_EJB;
+import com.ibm.ws.jpa.eclipselink.TestOLGH8950_Web;
+import com.ibm.ws.jpa.eclipselink.TestOLGH9018_EJB;
+import com.ibm.ws.jpa.eclipselink.TestOLGH9018_Web;
+import com.ibm.ws.jpa.eclipselink.TestOLGH9035_EJB;
+import com.ibm.ws.jpa.eclipselink.TestOLGH9035_Web;
```
Comment (dazey3) [inline dev/com.ibm.ws.jpa_eclipselink_fat/fat/src/com/ibm/ws/jpa/FATSuite.java:37]: Should these all be `com.ibm.ws.jpa.eclipselink.olgh` instead?

**Review by dazey3 — CHANGES_REQUESTED** [review]
Just one minor comment

**Review by dazey3 — APPROVED** [review]
LGTM

File: dev/com.ibm.ws.jpa_eclipselink_fat/fat/src/com/ibm/ws/jpa/FATSuite.java (hunk 1 above)
Comment (dazey3) [reply dev/com.ibm.ws.jpa_eclipselink_fat/fat/src/com/ibm/ws/jpa/FATSuite.java:37]: NVRM

**Pull request merged**